UIKitDZ/UIKitDZ/BookViewController.swift

```diff
@@ -0,0 +1,284 @@
+    private let textFontLabel = UILabel()
+    private let nightLabel = UILabel()
+
+    //MARK: - UIViewController
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        redColorButtonFunc()
+        greenColorButtonFunc()
+        blackColorButtonFunc()
+        yellowColorButtonFunc()
+        textSizeSliderFunc()
+        addWeightButtonFunc()
+        minusWeightButtonFunc()
+        fontPickerFunc()
+        nightSwitchFunc()
+        textSizeLabelFunc()
+        textFontLabelFunc()
+        nightLabelFunc()
+        bookTextViewFunc()
+        buttonShareFunc ()
+    }
+
+    private func bookTextViewFunc () {
+        bookTextView.frame = CGRect(x: 20, y: 220, width: self.view.bounds.width - 40,
+                                    height: self.view.bounds.height - 310)
+
+        bookTextView.text = """
+                                Повторить все что было в уроке
+                                Прочитать документацию по теме UITextView
+                                Сделать небольшую читалку (с заготовленным текстом)
+                                Функционал читалки:
+                                Изменение размера шрифта (через Slider), цвета букв (4
+                                кнопки разного цвета), толщину шрифта (две кнопки "a"
+                                "A")
+                                Через UIPicker сделать возможность выбора шрифта
+                                (на свой вкус 3-4 шрифта)
+                                Ночной режим (Switch) - менять фон (Светлый/темный)
+                                * Отправлять текст в другое приложение
+                                PS за пример можно взять iBook (ePub)
+                                Повторить все что было в уроке
+                                Прочитать документацию по теме UITextView
+                                Сделать небольшую читалку (с заготовленным текстом)
+                                Функционал читалки:
+                                Изменение размера шрифта (через Slider), цвета букв (4
+                                кнопки разного цвета), толщину шрифта (две кнопки "a"
+                                "A")
+                                Через UIPicker сделать возможность выбора шрифта
+                                (на свой вкус 3-4 шрифта)
+                                Ночной режим (Switch) - менять фон (Светлый/темный)
+                                * Отправлять текст в другое приложение
+                                PS за пример можно взять iBook (ePub)
+                            """
+        bookTextView.contentInset = UIEdgeInsets(top: 10, left: 0 , bottom: 0 , right: 0)
+        bookTextView.font = UIFont(name: "Arial", size: 8)
+        bookTextView.font = UIFont.systemFont(ofSize: 8, weight: UIFont.Weight(600))
+        bookTextView.backgroundColor = .gray
+        self.view.addSubview(bookTextView)
+    }
+
+    private func buttonShareFunc () {
```

Comment:
```suggestion
    private func buttonShare () {
```
убери везде Func в конце названия функции